.travis.yml

```diff
@@ -1,51 +1,43 @@
 # doc: http://about.travis-ci.org/docs/user/languages/scala/
```

Comment:
Not sure I'm happy for this comment to be removed - I recall spending _alot_ of time getting sbt to work with scalajs and documented results. All the more so, given that the settings have been altered that go against the comments, and indeed one day ago there was an issue to build failures based on these comments.